files/sidebars/cssref.yaml

```
@@ -598,9 +601,12 @@ l10n:
     Logical_properties: 逻辑属性
     Media_queries: 媒体查询
     Nesting: 嵌套样式规则
+    Overflow: 溢出
     Positioning: 定位
+    Scroll_anchoring: 滚动锚点定位
```

https://www.chinaw3c.org/category/w3c-rec/24/

```suggestion
    Scroll_anchoring: 滚动锚定
```